### DISCUSSION TITLE
The performance of different MCS in MIMO OFDM Transmissions over the CDL Channel Model?

### DISCUSSION THREAD

**Discussion by communicationDL1** [NVlabs/sionna#303] [Q&A]
I am committed to investigating the impact of different MCS on BLER, but I obtained unexpected erroneous results. Under the same 
Eb/No, higher order modulation and bit rates resulted in lower BLER, which is inconsistent with the situation measured by SNR. May I ask what could be the reason for this?
![1280X1280](https://github.com/NVlabs/sionna/assets/96966800/0fb88ec7-cbb8-49c6-9557-a96e163337f3)
 In this figure, 'CDL(4,0.3691)' denotes a 16QAM modulation scheme with a coding rate of 0.3691, 'CDL(6,0.455)' signifies a 64QAM modulation scheme with a coding rate of 0.455, and 'CDL(8,0.778)' indicates a 256QAM modulation scheme with a coding rate of 0.778.

**Comment by SebastianCa** [comment]
Hi,
it is difficult to provide detailed help without knowing your exact setup.

A few things to check:
- How is the performance at lower SNR (i.e., simulate the entire waterfall region)?
- Are the LDPC(?) codeword lengths constant? Keep in mind that they might change for higher order modulation if the number of symbols is kept constant. This could lead to coding gains due to longer codewords.
- Double check the Eb/No calculation and verify that all constellations have unit energy
